index.js

```diff
@@ -267,7 +267,7 @@ const Image = {
     // Upload the RAW image to disk, stripped of its extension
     // First check the token
     const sentToken = req.headers['x-token'];
-    const matches = req.url.match(/^\/(.*)\.([^.]+)$/);
+    const matches = req.path.match(/^\/(.*)\.([^.]+)$/);
```

Comment:
No this did work because of the format and constraints in the past. But yes it was a possible bug ;)

The url contains the full URL, including any query parameters. These were not used in the past, so never sent. The url was therefore equal to the path.
Since we now do accept query parameters, we want to save the file as `test.jpg` and not `test.jpg?x=0&y=4&width=100&height=400` but still as `test.jpg`. The path variable contains exactly this